udata/core/dataset/api.py

```diff
@@ -116,6 +117,24 @@ def __init__(self):
         self.parser.add_argument("topic", type=str, location="args")
         self.parser.add_argument("credit", type=str, location="args")
         self.parser.add_argument("dataservice", type=str, location="args")
+        self.parser.add_argument(
+            "archived",
+            type=boolean,
+            location="args",
+            help="If set to true, it will filter on archived datasets only. If set to false, it will exclude archived datasets. User must be authenticated and results are limited to user visibility",
+        )
+        self.parser.add_argument(
+            "deleted",
+            type=boolean,
+            location="args",
+            help="If set to true, it will filter on deleted datasets only. If set to false, it will exclude deleted datasets. User must be authenticated and results are limited to user visibility",
```

Comment:
Deleted datasets are not filtered by default?

Reply:
not anymore since [this PR](https://github.com/opendatateam/udata/pull/3214) :)